h2/src/test/org/h2/test/scripts/datatypes/enum.sql

```diff
@@ -107,9 +107,21 @@ alter table card alter column suit enum('hearts', 'clubs', 'spades', 'diamonds',
 drop table card;
 > ok
 
+CREATE TYPE my_number AS NUMBER;
+> ok
+
+alter type my_number ADD VALUE 'diamonds';
+> exception WRONG_OBJECT_TYPE
+
+drop type my_number;
+> ok
+
 --- ENUM as custom user data type
 
-create type CARD_SUIT as enum('hearts', 'clubs', 'spades', 'diamonds');
+create type CARD_SUIT as enum('hearts', 'clubs', 'spades');
+> ok
+
+alter type CARD_SUIT ADD VALUE 'diamonds';
```

Comment:
Add `@reconnect` somewhere after `ALTER TYPE` and test what happens with your enum after that.

Reply:
Currently it breaks with `@reconnec` between `CREATE TYPE` and `ALTER TYPE`.

Is it ok if I add another test case?